weave/weave_client.py

```diff
@@ -125,9 +125,11 @@ def _get_direct_ref(obj: Any) -> Optional[Ref]:
 
 
 def map_to_refs(obj: Any) -> Any:
-    ref = _get_direct_ref(obj)
-    if ref:
+    if ref := _get_direct_ref(obj):
         return ref
+    if isinstance(obj, ObjectRef):
+        obj = obj.get()
```

Comment:
why would we not return `obj` here?

Reply:
Thanks for catching, this was just for testing